### PR TITLE
Do not use presto-main in product tests

### PR DESCRIPTION
**Pull request by kokosing**
Do not use presto-main in product tests

### CODE REVIEW

**Comment by kokosing** [comment]
I recall it was used long time ago for query stats. 

Product tests are designed to black box end to end tests and they should not be allowed to use product code other that is designed to be used by users (jdbc, cli).

File: presto-product-tests/src/main/java/io/prestosql/tests/utils/QueryAssertions.java [hunk 1]
```diff
@@ -20,15 +20,32 @@
...
 
-import static io.prestosql.testing.assertions.Assert.assertEventually;
 import static java.lang.String.format;
 import static org.testng.Assert.fail;
 
 public final class QueryAssertions
 {
     public static void assertContainsEventually(Supplier<QueryResult> all, QueryResult expectedSubset, Duration timeout)
     {
-        assertEventually(timeout, () -> assertContains(all.get(), expectedSubset));
```
Comment (losipiuk) [inline presto-product-tests/src/main/java/io/prestosql/tests/utils/QueryAssertions.java:31]: why so?

Comment (kokosing) [reply presto-product-tests/src/main/java/io/prestosql/tests/utils/QueryAssertions.java:31]: It is copied from Presto main

Comment (kokosing) [reply presto-product-tests/src/main/java/io/prestosql/tests/utils/QueryAssertions.java:31]: Oh... It is copied below. So no need to inline it.

**Pull request merged**